cmd/kusk/manifests/api_server.yaml

```diff
@@ -113,7 +122,24 @@ spec:
         - image: kubeshop/kusk-gateway-api:v1.1.10
           imagePullPolicy: Always
           name: kusk-gateway-api
-      serviceAccountName: kusk-gateway-manager
+        - name: kusk-gateway-api-websocket
+          image: kubeshop/kusk-gateway-api-websocket:v.1.1.10
```

Comment:
I can't find the tag for the image in https://hub.docker.com/r/kubeshop/kusk-gateway-api-websocket/tags, only `latest` is present. Am I looking at the wrong page?

Reply:
The release CI in github actions will tag it as such. I want to keep it in lockstep with the api @mbana

Reply:
This tag is wrong actually, it should be v1.1.10